code/game/objects/items/charter.dm

```diff
@@ -93,3 +95,5 @@
 
 	if(!unlimited_uses)
 		used = TRUE
+
+#undef STATION_RENAME_TIME_LIMIT
```

Comment:
We shouldn't be defining things in random files, it should be in the defines. Also we should just have 2 separate values, it's not used in enough places. We might change 1 but not the other.